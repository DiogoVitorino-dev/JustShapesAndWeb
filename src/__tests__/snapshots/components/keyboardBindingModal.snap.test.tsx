import { render } from "test-utils";

import KeyboardBindingModal from "@/components/settings/keyboard/keyboardBindingModal";

describe("Keyboard Binding Modal - Snapshot component test", () => {
  it("should render correctly", () => {
    const tree = render(
      <KeyboardBindingModal data={{ command: "UP", key: "primary" }} />,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
