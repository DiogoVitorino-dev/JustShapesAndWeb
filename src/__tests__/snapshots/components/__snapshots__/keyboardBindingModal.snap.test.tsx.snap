// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Keyboard Binding Modal - Snapshot component test should render correctly 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    [
      {
        "flex": 1,
      },
      undefined,
    ]
  }
/>
`;
